modules/virt-upgrade-pathways.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * virt/upgrading-virt.adoc
+
+[id="virt-upgrade-pathways_{context}"]
+= Upgrade pathways for minor releases
+
+The upgrade pathway is different depending on the 2.4.z version of {VirtProductName} you have installed. 
+
+== Upgrading from 2.4.3 to 2.5.1
+
+You must first upgrade to 2.5.0 before you can upgrade the z-stream. You can then subsequently upgrade from 2.5.0 to 2.5.1.
+
+== Upgrading from 2.4.4 to 2.5.1
+
+You cannot currently upgrade from 2.4.4 to any 2.5 release. 
```

Comment:
The phrase "any 2.5 release" is currently true, but I wonder if we should say "2.5.0 or 2.5.1" instead, to be more exact.

Reply:
Good point